nautobot/extras/tests/test_jobs.py

```diff
@@ -302,6 +302,39 @@ def test_object_vars(self):
             self.assertEqual(info_log.log_object, "Role: role")
             self.assertEqual(job_result.data["output"], "\nNice Roles, bro.")
 
+    def test_optional_object_var(self):
+        """
+        Test that an optional Object variable field behaves as expected.
```

Comment:
Maybe also add a negative test to make sure that omitting a required variable fails/aborts as expected? I checked and I don't see such a test in the existing code.